lib/mongoHandler.js

```diff
@@ -162,6 +178,8 @@ MongoStore.prototype.initialise = function(resourceConfig) {
  */
 MongoStore.prototype.populate = function(callback) {
   var self = this;
+  if (!self._db) return;
```

Comment:
`populate` is reserved for the test suite, not end users. This line prevents the example server crashing out when proving "use case 1".